docs/src/quickstart/index.md

```diff
@@ -0,0 +1,307 @@
+# (#ref-qs) Installation and Setup
+
+This quickstart guide outlines the step-by-step instructions for getting started programming in Reach for Windows, Linux, and Mac.
+
+# (#ref-qs-win) Windows
+
+Reach requires [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker](https://www.docker.com/get-started), and [Docker Compose](https://docs.docker.com/compose/install/).
+
+## (#ref-qs-win-prereqs) Prerequisites
+
++Windows 10 installed with version 2004 or higher
+
+## (#ref-qs-win-install) Installation
+
+Click the Windows icon, type `CMD`, and then click `Run as Administrator`.
+There are a number of commands that need to be run to get Windows ready for Reach.
+
+In the Command Prompt window, run:
+
+``` cmd
+$ wsl --install -d ubuntu
+```
+
+This installs Ubuntu WSL, which is used via command line later.
+
+In the Command Prompt window, run:
+
+``` cmd
+$ wsl --set-version ubuntu 2
+``` 
+
+This sets the WSL to version 2, which is required by Docker and Reach.
+
+Next, download and install [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+:::note
+Make sure to follow Docker's instructions for installation and configuration.
+Those instructions are subject to change and are outside of our control.
+:::
+
+Click the `Settings`(gear) icon along the top of the Docker app.
+
+![Settings icon in Docker`](/qs/settings-icon.png)
+
+Verify that `WSL 2.0` is checked, as well as `Use Docker Compose V2`.
+
+![WSL 2 settings in Docker`](/qs/wsl-2.png)
+![Docker Compose in Docker`](/qs/docker-compose.png)
+
+Click `Resources` in the left-hand menu, and check that `Enable integration with additional distros` is checked, and that `Ubuntu` is selected.
+
+![WSL integration in Docker`](/qs/wsl-integrate.png)
+
+Double-click the Ubuntu icon in the Windows Start-up menu to open the Ubuntu terminal.
```

Comment:
```suggestion
Click the Ubuntu icon in the Windows Start-up menu to open the Ubuntu terminal.
```

I believe just a single click is needed, correct? Double-clicks might be necessary for desktop icons.